### DISCUSSION TITLE
[v3] Unique Validation Rule with #Rule attribute

### DISCUSSION THREAD

**Discussion by c-fitzmaurice** [livewire/livewire#6397] [7. Bugs (archived)]
### Livewire version

v3.0.0-beta.9

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Install Volt and Folio
2. Use a Livewire Form with `unique` validation rule

## Example Code:

```php
// register.blade.php volt component
<?php
use App\Livewire\Forms\Register;

form(Register::class);

$submit = fn () => $this->form->submit();
?>

<form wire:submit="submit">
    <input wire:model="form.name">

    <input wire:model="form.email">

    <input wire:model="form.password">

    <button type="submit">Save</button>
</form>

```

```php
// app/Livewire/Forms/Register.php Livewire Form

use App\Models\User;

class Register extends Form
{
    #[Rule('required|string')]
    public ?string $name;

    #[Rule(['required', 'string', 'email', 'unique:'.User::class])]
    public ?string $email;

    #[Rule(['required'])]
    public ?string $password;

    public function submit(): RedirectResponse
    {
        $this->validate();

        ...
    }
}
```

## Error

```sql
select count(*) as aggregate from `users` where `form`.`email` = hi@example.com
```

### Additional information

Other question; is it possibel to use Laravel's `\Illuminate\Validation\Rules\Password::defaults()` password defaults rule in the attribute validation?

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@c-fitzmaurice you need to also specify the column name otherwise it will try to use `form.email` as the column name. So like `unique:'.User::class.',email`.

Hope this helps!

**Reply by c-fitzmaurice** [reply]
Oh man, thanks!

🤦🏼‍♂️ Closing cause this is not a Livewire concern, I was being an airhead.

**Reply by joshhanley** [reply]
@c-fitzmaurice no worries! 🙂
